### ISSUE TITLE
Reference to eslint formatter left in webpack config when not including eslint in setup

### ISSUE THREAD

**Issue by hultberg** [vuejs-templates/webpack#486]
Hello,

When answering no the question about eslint, an require is left in the webpack.base.conf.js file, making `npm run dev` fail with:
```
Error: Cannot find module 'eslint-friendly-formatter'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/???/projects/???/build/webpack.base.conf.js:5:31)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```
PR: #487

**Comment by cschweda** [comment]
Yep. Had to install this '--save-dev' for the build to work.

**Comment by posva** [comment]
@kazupon can be closed because of 0b8051f

**Comment by kazupon** [comment]
@posva thanks
